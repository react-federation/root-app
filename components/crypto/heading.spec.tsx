import { ChakraProvider } from '@chakra-ui/react'
import setup from 'utils/test/setup'
import Heading from './heading'

describe('Heading', () => {
  it('should match snapshot', () => {
    const { asFragment } = setup(Heading, { image: { small: '' }, name: '', symbol: '' }, ChakraProvider)

    expect(asFragment()).toMatchSnapshot()
  })
})
