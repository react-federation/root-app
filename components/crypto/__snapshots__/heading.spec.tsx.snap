// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should match snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-1 {
  width: 32px;
  height: 32px;
  margin-right: var(--chakra-space-2);
}

.emotion-2 {
  font-family: var(--chakra-fonts-heading);
  font-weight: var(--chakra-fontWeights-bold);
  font-size: var(--chakra-fontSizes-3xl);
  line-height: 1.33;
}

@media screen and (min-width: 48em) {
  .emotion-2 {
    font-size: var(--chakra-fontSizes-4xl);
    line-height: 1.2;
  }
}

.emotion-3 {
  color: var(--chakra-colors-purple-600);
}

<div
    class="emotion-0"
  >
    <img
      alt=" logo"
      class="chakra-image__placeholder emotion-1"
    />
    <h2
      class="chakra-heading emotion-2"
    >
       
      <span
        class="chakra-text emotion-3"
      >
        ()
      </span>
    </h2>
  </div>
</DocumentFragment>
`;
